amaranth-chrome-ext/src/AmaranthUtil.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Utility class with helpful functions for the Amaranth Chrome extension.
+ * All methods are static.
```

Comment:
This comment as is should be removed. Unless you're trying to state that EVERY method in the class should be static, in which case directly state that along with a "why".

Reply:
They definitely should all be static. This utility class is really just a wrapper for some (pure) helper functions and should contain no internal state. I'll clarify.

Reply:
Fixed in https://github.com/googleinterns/amaranth/pull/44/commits/fd2c438bb10cd81257c0dec0b6bb80c29f6c5324